uniffi_checksum_derive/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "uniffi_checksum_derive"
```

Comment:
Why a new crate instead of putting the derive macro into `uniffi_macros`?

Reply:
Because it's not a macro meant to be used as part of the public API, but more importantly, because uniffi_macros depends on uniffi_meta, which would make it circular, although technically speaking, it's also possible to not make uniffi_meta reexport the derive, but that's less convenient.